src/main/resources/application.properties

```diff
@@ -45,8 +45,6 @@ integration.invoicecache.readTimeout=20
 # Oauth2
 spring.security.oauth2.client.registration.invoicecache.authorization-grant-type=client_credentials
 spring.security.oauth2.client.registration.invoicecache.provider=invoicecache
-# Circuit breaker settings
```

Comment:
IMHO I think it's easier to keep all settings for the integration together in one place